build/pack.ps1

```diff
@@ -111,14 +111,22 @@ Pack-Nuget '../src/Core/Core.csproj'
 Write-Host "##[info]Packing IQ# tool..."
 Pack-Nuget '../src/Tool/Tool.csproj'
 
-Write-Host "##[info]Packing IQ# as self-contained executables."
-Pack-Exe "../src/Tool/Tool.csproj" -Runtime win10-x64
-Pack-Exe "../src/Tool/Tool.csproj" -Runtime osx-x64
-Pack-Exe "../src/Tool/Tool.csproj" -Runtime linux-x64
-
-Write-Host "##[info]Packing Python wheel..."
-python --version
-Pack-Wheel '../src/Python/'
+if ($Env:ENABLE_PYTHON -eq "false") {
+    Write-Host "##vso[task.logissue type=warning;]Skipping Creating Python packages. Env:ENABLE_PYTHON was set to 'false'."
+} else {
+    Write-Host "##[info]Packing IQ# as self-contained executables."
+    Pack-Exe "../src/Tool/Tool.csproj" -Runtime win10-x64
+    Pack-Exe "../src/Tool/Tool.csproj" -Runtime osx-x64
+    Pack-Exe "../src/Tool/Tool.csproj" -Runtime linux-x64
```

Comment:
These steps are part of the conda feature, and aren't included in Python wheels. I'd suggest moving these to be  guarded by `$Env:ENABLE_CONDA` instead, perhaps?

Reply:
Good point.